install/installer/pkg/components/usage/configmap.go

```diff
@@ -36,6 +36,12 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 			ForUsers:            1_000_000_000,
 			MinForUsersOnStripe: 0,
 		},
+
+		// This is the default configuration, used in self-hosted installs.
+		// For SaaS, we explicitly configure it based on workspace classes below.
+		CreditsPerMinuteByWorkspaceClass: map[string]float64{
+			"default": 0.1666666667,
```

Comment:
Is the credits value significant? If not, adding a comment to that effect might be worthwhile.

Reply:
It is significant. At least the `default` class has to be defined. We could define it with a value of 0, but that risks actually making a mistake and not charging for instances at all in our SaaS..